libraries/src/Console/SetConfigurationCommand.php

```diff
@@ -445,7 +445,9 @@ public function sanitizeOptions(array $options): array
         foreach ($options as $key => $value) {
             $value = $value === 'false' ? false : $value;
             $value = $value === 'true' ? true : $value;
-
+            
+            $value = ctype_digit((string) $value) ? (int) $value : $value;
+            
```

Comment:
```suggestion
            $value = ctype_digit((string) $value) ? (int) $value : $value;

```
PHPCS still complaining about white space in empty line.